modules/http/src/main/scala/higherkindness/mu/http/implicits.scala

```diff
@@ -96,17 +96,18 @@ object implicits {
       stream.attempt.map(_.bimap(_.toUnexpected, identity).asJson)
   }
 
-  implicit class FResponseOps[F[_]: Sync](private val response: F[Response[F]])
-      extends Http4sDsl[F] {
+  implicit class FResponseOps[F[_]](private val response: F[Response[F]]) extends Http4sDsl[F] {
 
-    def adaptErrors: F[Response[F]] =
+    def adaptErrors(implicit F: ApplicativeThrow[F]): F[Response[F]] =
```

Comment:
📝 Putting the implicit requirement on the extension method instead of on the ops class generally leads to nicer compile errors for callers who are missing the appropriate instance in scope. At the ops constructor, they get a "no such method" error, whereas on the extension they get a "missing implicit parameter" error. I ran into that while refactoring one call site